swagger2.0/oic.sec.ael.swagger.json

```diff
@@ -258,6 +226,25 @@
 			"required": [
 				"events", "usedspace", "maxspace"
 			]
+		},
+		"Ael-Update": {
+			"type": "object",
+			"properties": {
+				"categoryfilter": {
+					"description" : "This value decides what categories of AEEs are to be logged. Meaning of each bit: 0x01 (Access Control), 0x02 (Onboarding), 0x04 (Device), 0x08 (Authentication), 0x10 (SVR Modification), 0x20 (Cloud), 0x40 (Communication). e.g.) if categoryfilter == 0xff: log all events of all categories,  e.g.) if categoryfilter == 0x03: log all events of 'AC (== 0x01)' and 'OB (==0x02)' categories ",
+					"type": "integer", 
+					"default": 256
+				},
+				"priorityfilter": {
+					"description": "The AEEs whose 'priority' values are equal to or smaller than this value are logged. A smaller value means a higher priority. Meaning of each value: 0 (CRIT), 1 (ERR), 2 (WARN), 3 (INFO), 4 (DEBUG). e.g.) if priorityfilter is set to DEBUG (==4) all AEEs will be logged, e.g.) if priorityfilter is set to 1, CRIT (==0) and ERR (==1) AEEs will be logged ",
+					"type": "integer",
+					"default": 4,
+					"enum": [
+						0, 1, 2, 3, 4
+					]
+				}
+			},
+			"required": []
```

Comment:
Generally, when there is no "required" Properties then this line is left out completely.  It might make sense to check with the DMWG to see what the BCP is for this because I'm not sure.

Reply:
Makes sense